src/android/InAppBrowser.java

```diff
@@ -262,7 +263,12 @@ else if (action.equals("loadAfterBeforeload")) {
                 @SuppressLint("NewApi")
                 @Override
                 public void run() {
-                    ((InAppBrowserClient)inAppWebView.getWebViewClient()).waitForBeforeload = false;
+                    if (android.os.Build.VERSION.SDK_INT <= android.os.Build.VERSION_CODES.N_MR1) {
```

Comment:
I would say `android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.O`, since the Android docs mention "added in API level 26".

Reply:
Better indeed. Done.